boruta/test/test.sh

```diff
@@ -26,7 +26,7 @@ run_bigmler execute --scripts .build/scripts --inputs test_inputs.json \
                     --output-dir cmd/results
 # check the outputs
 declare file="cmd/results/whizzml_results.json"
-declare regex="\"outputs\": \[\[\"feature-selected-dataset\", "
+declare regex="\"outputs\": \[\[\"feature-selected-dataset\", .*\[\"results\", {"
```

Comment:
maybe \{ instead of { ?

Reply:
I ran it, and I've done it again and it doesn't report me any problem 😐 I will escape the characters, try it, and commit it.